src/Analyzers/Core/CodeFixes/PopulateSwitch/AbstractPopulateSwitchStatementCodeFixProvider.cs

```diff
@@ -57,7 +57,11 @@ protected AbstractPopulateSwitchStatementCodeFixProvider()
         }
 
         protected sealed override ITypeSymbol GetSwitchType(ISwitchOperation switchOperation)
-            => switchOperation.Value.Type;
+        {
+            var type = switchOperation.Value.Type;
+            RoslynDebug.AssertNotNull(type);
+            return type;
+        }
```

Comment:
💭 Maybe better to do something like this?

```suggestion
            => switchOperation.Value.Type ?? throw ExceptionUtilities.Unreachable;
```